### DISCUSSION TITLE
3.4.0 Released

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#2671] [General]
Announcement posted here: https://www.oqtane.org/blog/!/56/oqtane-3-4-0-released

**Comment by W6HBR** [comment]
Just want to say thanks to all the contributors that make these releases possible. Thank you!

**Comment by PavelVsl** [comment]
I have just upgraded my project from version 3.2 to the new version of Oqtane Framework without issues. The only thing I had to do was replace Permissions with PermissionList. Thanks to @sbwalker and all contributors for this great piece of software.

**Comment by sbwalker** [comment]
@chlupac I knew you would be happy with the permissions improvements - you called them out quite a while ago as an area that was in need of some refactoring.

**Comment by sbwalker** [comment]
A few unfortunate issues were discovered in the 3.4 release within the past couple weeks... a 3.4.2 stabilization release will be published early next week which addresses these items. I really appreciate the efforts of the community in testing the framework and raising issues so they can be remediated as quickly as possible.

**Reply by leigh-pointer** [reply]
@sbwalker truly appreciated, thank you!

**Comment by sbwalker** [comment]
3.4.2 was released today
